packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2725,15 +2727,8 @@ test('manager notifiers', async t => {
     totalDebt: { value: DEBT1 },
   });
   m.addDebt(DEBT1);
-  const periods = 5n;
-  // FIXME test result relies on awaiting each tick
-  // timer.tickN() awaits only the last tick
-  for (let i = 0; i < periods; i += 1) {
-    // eslint-disable-next-line no-await-in-loop
-    await manualTimer.tick();
-  }
-  // wait to let interest charging catch up
-  await eventLoopIteration();
+  const periods = 5;
+  await manualTimer.tickN(periods);
```

Comment:
`periods` had been used for `t.is(interestAccrued, periods * 2n)` but not anymore and I'm not confident the math works out that cleanly for arbitrary N.

if you feel like it,
```suggestion
  await manualTimer.tickN(5);
```

Reply:
done